libcontainerd/container_windows.go

```diff
@@ -60,12 +61,27 @@ func (ctr *container) start() error {
 	// is only created if it we're not -t.
 	var pid uint32
 	var stdout, stderr io.ReadCloser
-	pid, iopipe.Stdin, stdout, stderr, err = hcsshim.CreateProcessInComputeSystem(
-		ctr.containerID,
-		true,
-		true,
-		!ctr.ociSpec.Process.Terminal,
-		createProcessParms)
+
+	var retry = 0
+	for {
+		pid, iopipe.Stdin, stdout, stderr, err = hcsshim.CreateProcessInComputeSystem(
+			ctr.containerID,
+			true,
+			true,
+			!ctr.ociSpec.Process.Terminal,
+			createProcessParms)
+		if err != nil && strings.Contains(err.Error(), "HCSShim::CreateComputeSystem failed in Win32: The system cannot find the file specified. (0x2)") {
```

Comment:
It'd be nice if this was a typed error that we can compare against instead of checking the error string.